op_crates/websocket/01_websocket.js

```diff
@@ -125,7 +125,12 @@
               const event = new CloseEvent("close");
               event.target = this;
               this.dispatchEvent(event);
-              core.close(this.#rid);
+              try {
+                core.close(this.#rid);
+              } catch (err) {
+                // Ignore error if the socket has already been closed.
+                if (!(err instanceof Deno.errors.BadResource)) throw err;
+              }
```

Comment:
Nitpick: make this bit a helper function?

Reply:
Done now